app/screens/node/Node.tsx

```diff
@@ -341,7 +341,7 @@ const Node = ({ history, location }: Props) => {
   const handleSetupSmesher = () => {
     return eventsService
       .switchApiProvider(LOCAL_NODE_API_URL)
-      .then(() => history.push('/auth', { redirect: '/main/node-setup' }))
+      .then(() => history.push('/auth/unlock', { redirect: '/main/node-setup' }))
```

Comment:
is it possible to create constants for routes that we have, even only in this PR, to keep names and paths not in memory and take from a constant near routing.

Reply:
Actually, I'd like to refactor all this mess around `react-router` and probably start using something with redux :)
But I've replaced all these hard-coded lines with enums. You can check it out ;)